lib/services/api_handler.dart

```diff
@@ -146,6 +174,13 @@ abstract class AlertAPI {
   /// else went wrong
   Future<ServerSettings> fetchServerSettings({String overrideUrl});
 
+  /// fetch the for webpush needed vapid key from the server
+  ///
+  /// returns the vapid key as String if it was successfully fetched
+  ///
+  /// Throws an exception if something went wrong
```

Comment:
```suggestion
  /// Throws a VapidKeyException if something went wrong
```

Always be specific in documenting what is thrown so API users don't have to dig into the code to see what can happen.